Assets/Samples/ProjectWideActionsTest/ProjectWideActionsTest.cs

```diff
@@ -1,63 +1,42 @@
 #if UNITY_INPUT_SYSTEM_PROJECT_WIDE_ACTIONS
 
 using UnityEngine;
-using UnityEngine.InputSystem;
 
 public class NewBehaviourScript : MonoBehaviour
 {
     [SerializeField] public GameObject cube;
 
-    InputAction move;
-    InputAction look;
-    InputAction attack;
-    InputAction jump;
-    InputAction interact;
-    InputAction next;
-    InputAction previous;
-    InputAction sprint;
-    InputAction crouch;
-
     // Start is called before the first frame update
     void Start()
     {
-        // Project-Wide Actions
```

Comment:
I'm thinking It would be good to keep one example of the explicit InputSystem.actions FindAction route for any user wanting to lookup by string for some reason of dynamic access, but this is the more advanced option